sdk/openai/Azure.AI.OpenAI/src/Custom/ChatCompletions.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Azure.Core;
+
+namespace Azure.AI.OpenAI
+{
+    /// <summary>
+    /// A representation of a full response to a chat completions request.
+    /// </summary>
+    /// <remarks>
+    /// Chat completions can contain one or more chat choices, each holding a message with string content and an
+    /// associated role. When requesting more than one completion, a corresponding number of additional choices will
+    /// appear in the chat completions result.
+    /// </remarks>
+    public partial class ChatCompletions
+    {
+        internal ChatCompletions(string id, int? created, IReadOnlyList<ChatChoice> choices, CompletionsUsage usage)
+        {
+            Id = id;
+            Created = TimeConverters.DateTimeFromUnixEpoch(created.Value);
+            Choices = choices.ToList();
+            Usage = usage;
+        }
+
+        /// <summary>
+        /// Gets a unique identifier associated with a chat completions response.
+        /// </summary>
+        public string Id { get; }
+        /// <summary>
+        /// Gets the UTC timestamp at which this chat completions response was generated.
+        /// </summary>
+        public DateTime Created { get; }
```

Comment:
I'd love to follow up with you on this one for the next PR -- I actually tried to define `created` as a `zonedDateTime` in the typespec, but the particular semantics of the field (seconds since the UNIX epoch at the beginning of 1970) didn't lend themselves to the default handling.

If we can manage it entirely via the codegen process, that'll be great; but even if we can't, agreed that aligning on the preferred type is better.